data/hai ships.txt

```diff
@@ -94,6 +151,106 @@ ship "Flea"
 
 
 
+ship "Geocoris"
+	sprite "ship/hai geocoris"
+	attributes
+		category "Heavy Freighter"
+		"cost" 12540000
+		"shields" 13300
+		"hull" 6200
+		"required crew" 15
+		"bunks" 56
+		"mass" 810
+		"drag" 17.1
+		"heat dissipation" 0.45
+		"fuel capacity" 800
+		"cargo space" 761
+		"outfit space" 582
+		"weapon capacity" 172
+		"engine capacity" 94
+		Weapon:
+			Blast Radius: 300
+			Hit Force: 4500
+			Shield damage: 3000
+			Hull damage: 1500
+	outfits
+		Hyperdrive
+		"Hai Williwaw Cooling" 5
+		"Hai Valley Batteries"
+		"Pulse Turret" 2
+		"Pulse Cannon" 2
+		"Bullfrog Anti-Missile" 5
+		"Hai Diamond Regenerator"
+		`"Bondir" Atomic Steering`
+		"Geode Reactor" 2
+		`"Biroo" Atomic Thruster`
+	engine 64 77 1
+	engine -64 77 1
+	turret 47 -73 "Pulse Turret"
+	turret -47 -73 "Pulse Turret"
+	turret 37 6.5 "Bullfrog Anti-Missile"
+	turret -37 6.5 "Bullfrog Anti-Missile"
+	turret 0 63 "Bullfrog Anti-Missile"
+	turret 65 25 "Bullfrog Anti-Missile"
+	turret -65 25 "Bullfrog Anti-Missile"
+	gun 25 -109 "Pulse Cannon"
+	gun -25 -109 "Pulse Cannon"
+	explode "tiny explosion" 50
+	explode "small explosion" 50
+	explode "medium explosion" 50
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large"
+	description "Just a few years after the first humans discovered Hai space, the Hai knew they would eventually need a bigger freighter to match the larger human ones. What they did not expect, however, is that the result would surpass anything humanity has to offer, which is why it became such a popular ship amongst Hai and humans alike."
+	description "At first the Geocorsis started as a modified Water Bug, and you're sure you see a vague sign of it somewhere..."
+
+
+
+ship "Grasshopper"
+	sprite "ship/hai grasshopper"
+	attributes
+		category "Interceptor"
+		"cost" 684000
+		"shields" 3300
+		"hull" 600
+		"required crew" 1
+		"bunks" 5
+		"mass" 95
+		"drag" 1.5
+		"heat dissipation" .85
+		"fuel capacity" 300
+		"cargo space"  29
+		"outfit space"  169
+		"weapon capacity" 38
+		"engine capacity" 61
+		weapon
+			"blast radius" 36
+			"shield damage" 180
+			"hull damage" 540
+			"hit force" 360
+	outfits
+		Hyperdrive
+		"Pulse Cannon"
+		"Hai Tracker Pod"
+		"Hai Chasm Batteries"
+		"Hai Williwaw Cooling" 2
+		"Hai Corundum Regenerator"
+		"Hai Tracker" 56
+		`"Benga" Atomic Thruster`
+		"Pebble Core"
+		`"Benga" Atomic Steering`
+	engine 19 33 1
+	engine -19 33 1
+	gun 11 -35 "Hai Tracker Pod"
+	gun -11 -35 "Pulse Cannon"
+	explode "tiny explosion" 10
+	explode "small explosion" 20
+	explode "medium explosion" 5
+	"final explode" "final explosion small"
+	description "Designed by the Hai to serve as a fast reconnaissance craft, it has been adopted by thrill-seekers and miners alike. At the same time, it is often used by Hai captains who are looking for effective protection at a low cost."
```

Comment:
`fast reconnaissance craft, it has been adopted` 
Use the ship's name instead of `it` here.